templates/cassiopeia/offline.php

```diff
@@ -39,16 +59,17 @@
 
 if ($this->params->get('logoFile'))
 {
-	$logo = '<img src="' . Uri::root() . $this->params->get('logoFile') . '" alt="' . $sitename . '">';
+	$logo = '<img src="' . Uri::root() . htmlspecialchars($this->params->get('logoFile'), ENT_QUOTES) . '" alt="' . $sitename . '">';
```

Comment:
In the error.php above, no `htmlspecialchars` is used for the `$this->params->get('logoFile')`. So what is right? This here, or error.php?

Reply:
I used the same syntax as in index.php, so I think the htmlspecialchars is correct

Reply:
Shouldn't it be used then in the error.php, too?

Reply:
Corrected in last commit